plugins/scaffolder-common/src/Template.v1beta3.schema.json

```diff
@@ -0,0 +1,186 @@
+                    "description": "The ID of the step, which can be used to refer to its outputs."
+                  },
+                  "name": {
+                    "type": "string",
+                    "description": "The name of the step, which will be displayed in the UI during the scaffolding process."
+                  },
+                  "action": {
+                    "type": "string",
+                    "description": "The name of the action to execute."
+                  },
+                  "input": {
+                    "type": "object",
+                    "description": "A templated object describing the inputs to the action."
+                  },
+                  "if": {
+                    "type": ["string", "boolean"],
+                    "description": "A templated condition that skips the step when evaluated to false. If the condition is true or not defined, the step is executed. The condition is true, if the input is not `false`, `undefined`, `null`, `\"\"`, `0`, or `[]`."
+                  }
+                }
+              }
+            },
+            "output": {
+              "type": "object",
+              "description": "A templated object describing the outputs of the scaffolding task.",
+              "properties": {
+                "links": {
+                  "type": "array",
+                  "description": "A list of external hyperlinks, typically pointing to resources created or updated by the template",
+                  "items": {
+                    "type": "object",
+                    "required": [],
+                    "properties": {
+                      "url": {
+                        "type": "string",
+                        "description": "A url in a standard uri format.",
+                        "examples": ["https://github.com/my-org/my-new-repo"],
+                        "minLength": 1
+                      },
+                      "entityRef": {
+                        "type": "string",
+                        "description": "An entity reference to an entity in the catalog.",
+                        "examples": ["Component:default/my-app"],
+                        "minLength": 1
+                      },
+                      "title": {
+                        "type": "string",
+                        "description": "A user friendly display name for the link.",
+                        "examples": ["View new repo"],
+                        "minLength": 1
+                      },
+                      "icon": {
+                        "type": "string",
+                        "description": "A key representing a visual icon to be displayed in the UI.",
+                        "examples": ["dashboard"],
+                        "minLength": 1
+                      }
+                    }
+                  }
+                }
+              },
+              "additionalProperties": {
+                "type": "string"
```

Comment:
is it true still that outputs can only be strings? just double checking :)

Reply:
Definitely not, since the `"links"` above is an array of objects, @benjdlambert? 😅

Reply:
In the Template they are strings though.


```
outputs:
    templatePath: ${{ parameters.lol }}
```